src/MiNET/MiNET/Worlds/LevelDbProvider.cs

```diff
@@ -189,7 +189,9 @@ public ChunkColumn GetChunk(ChunkCoordinates coordinates, IWorldGenerator genera
 				if (flatDataBytes != null)
 				{
 					Buffer.BlockCopy(flatDataBytes.AsSpan().Slice(0, 512).ToArray(), 0, chunkColumn.height, 0, 512);
-					chunkColumn.biomeId = flatDataBytes.AsSpan().Slice(512, 256).ToArray();
+
+					// TODO - 1.20 - update
```

Comment:
Any chance you could look into loading the biome id's in still?

Reply:
This should be supported, since the client is currently working with 3d chunks.

Reply:
should it be active?